src/pages/UserInfo.jsx

```diff
@@ -1,32 +1,51 @@
-import React from "react";
-import { useSelector } from "react-redux";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { motion } from "framer-motion";
+import Link from "next/link";
+import { setDay, setModalDefault, setModalNight, setNight, setTextDefault, setTextNight } from "@/slices/dashboardSlice";
 const UserInfo = () => {
-
+  const [isDay, setIsDay] = useState(true);
```

Comment:
can you explain this piece of code here?
and why you kept the state as true by default?

Reply:
this is for the day and night theme and by default, it is set to day